### PR TITLE
Updating the tech test to include Apps.

### PR DESCRIPTION
**Pull request by johnriot**
The tech test wording is a little Web-specific, we've made it a bit more general.
We've added a hint to allow a developer to use the public foursquare token (saves them wasting some time doing API key generation).

### CODE REVIEW

**Comment by johnriot** [comment]
@mquailey @marcellomascia @markstickley : Making the tech-test readme a little more general to include / not confuse app developers. Please review :)

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ You are to complete as much as you see fit to show us what you are capable of.
 
 ## The Scenario
 
-We have a website, were we would like to include a [Foursquare integration](https://developer.foursquare.com/) in the future, as part of this task you have been chosen to create a demo showing how we can integrate the API to show something useful.
+We have a website / apps, where we would like to include a [Foursquare integration](https://developer.foursquare.com/) in the future. As part of this task you have been chosen to create a demo showing how we can integrate the API to show something useful.
```
Comment (markstickley) [inline README.md]: app, singular?

File: README.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ You are then free to display the results as you see fit.
 
 ## Rules
 
-*   We think that this is easiest to complete using HTML/JS/CSS but **you are able to use any language, platform, framework or libraries that you think are useful.**
+*   **You are able to use any language, platform, framework or libraries that you think are useful.** For web developers, we think that this is easiest to complete using HTML/JS/CSS. For app developers we expect it to be completed in a native language for your platform (Java, Swift etc).
```
Comment (markstickley) [inline README.md]: If we are recruiting Swift devs that's cool, otherwise we could maybe list Objective-C as well.

**Comment by markstickley** [comment]
Added a few suggestions. Another suggestion would be to expand upon the bit which says "We must be able to run your code" saying they should make it as easy as humanly possible to run said code. Instructions, included libraries, scripts. All very handy.

**Comment by johnriot** [comment]
@markstickley Thanks for the comments, pushed a new commit to address them :)

**Comment by markstickley** [comment]
👍🏼

**Comment by marcellomascia** [comment]
:+1:

**Comment by mquailey** [comment]
Looks good to me :+1:

**Pull request merged**